cache.py

```diff
@@ -8,6 +8,12 @@ def purge(*args):
     for path in args:
         run("curl -s -I -X PURGE http://localhost:7999%s | grep '200 Purged'" % path.strip())
 
+@task
+@roles('class-cache', 'class-draft_cache')
+def purge_all():
+    "Purge everything from varnish. This is done by restarting all varnish processes."
+    restart()
```

Comment:
This scares me a bit because of its ability to immediately increase load on the app servers (I have no data to back that up, it's just a feeling).